saq/worker.py

```diff
@@ -280,6 +281,7 @@ def import_settings(settings):
 
 
 def start(settings, web=False, extra_web_settings=None, port=8080):
+    sys.path.insert(0, os.getcwd())  # fix: https://github.com/tobymao/saq/issues/26
```

Comment:
This feels risky to me. We don't actually know what directory saq is being invoked from.

_If_ we add something like this, it should probably be in https://github.com/tobymao/saq/blob/master/saq/__main__.py